quiz-microsevice/cli/actions/regenerate-client-credentials.js

```diff
@@ -0,0 +1,17 @@
+const { default: mongoose } = require("mongoose");
+const { clientService } = require("../../business-logic/services");
+const ValidationError = require('../../business-logic/errors/common/validation-error')
+
+const regenerateClientCredentials = async (id) => {
+    
+    // const vaildId = mongoose.Types.ObjectId.isValid(id)
```

Comment:
This is good but do it within business-logic layer.

Reply:
I won't use it, anyway.

Reply:
It can't be in the business logic; I've tested it, and this process only works here.

But anyway I won't use it.